boards/arm/nucleo_l552ze_q/CMakeLists.txt

```diff
@@ -27,36 +28,36 @@ if (CONFIG_BUILD_WITH_TFM)
 	set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
 
 		#Sign secure binary image with public key
-		COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/imgtool.py
-		ARGS sign
-			 --layout ${PREPROCESSED_FILE}_s.c
+		COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/wrapper/wrapper.py
+			 --layout ${PREPROCESSED_FILE_S}
 			 -k ${CONFIG_TFM_KEY_FILE_S}
 			 --public-key-format ${TFM_PUBLIC_KEY_FORMAT}
 			 --align 1
 			 -v ${TFM_IMAGE_VERSION_S}
+			 --pad
+			 --pad-header
 			 ${ADD_NS_IMAGE_MIN_VER}
-			 ${ADD_SECURITY_COUNTER_S}
+			 -s auto
 			 -H 0x400
-			 ${CMAKE_BINARY_DIR}/tfm/install/outputs/STM_NUCLEO_L552ZE_Q/tfm_s.bin
+			 ${CMAKE_BINARY_DIR}/tfm/install/outputs/MPS2/AN521/tfm_s.bin
```

Comment:
I guess this change is not correct, but I'm not sure if it should be `stm/nucleo_l552ze_q` of  the existing line was fine

Reply:
Looks like we don't actually need custom vendor/board paths, binaries are copied to ./build/tfm/bin/
This could be used further to unify the merge script.

Reply:
That's probably true @agansari, we can revisit this as part of the cleanup (something along what @anangl had done in https://github.com/zephyrproject-rtos/zephyr/pull/28806)

I'd just fix this for now, since I want to separate the upmerge from the cleanup 👍

Reply:
This is fixed, now, @erwango